src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js

```diff
@@ -40,12 +44,13 @@ export const uiSchema = {
     ),
     'ui:options': {
       widgetClassNames: 'usa-input-medium',
+      hideEmptyValueInReview: true,
     },
     'ui:errorMessages': {
       pattern: 'Please enter a valid VA file number',
     },
   },
   dateOfBirth: merge(currentOrPastDateUI('Date of birth'), {
-    'ui:required': () => !hasSession(),
+    'ui:required': formData => !hasSession() || formData.loa !== 3,
```

Comment:
Wouldn't it be more consistent to include the `LOA_LEVEL_REQUIRED` constant in this file as well? 😺

Reply:
..yes.. 😅